docs/1-introduction/1-1-getting-started/1-1-3-hello-triangle.md

```diff
@@ -14,19 +21,29 @@ All the steps in the graphics pipeline go from top to bottom and are shown below
 As you can see, each stage in the pipeline takes in the previous stage's output as the input, the rectangle blocks are pipeline stages that are not programmable but are configurable, while the rounded rectangle blocks are stages that are fully programmable.
 To draw most of the things throughout this series we'll mostly need these stages: Input Assembler, Vertex Shader and the Pixel Shader, Output Merger.
 
-The Vertex and Pixel shaders are fully programmable, and we'll write a very basic program for them.
+The Vertex and Pixel shaders are fully programmable and we'll write a very basic program for them.
 
-The other two stages are not programmable, but they are fairly easy to understand and configure, the **Input Assembler** is responsible for processing the vertices in an eventual vertex buffer into the primitive of our choice, which is of course, triangles in our case, and sending this processed output to the Vertex Shader. The **Output Merger** instead is responsible for combining the values written by the pixel shader, may that be depth, color or other things, into the one or more render targets that we provide to the OM, we only have one render target for now.
+The other two stages are not programmable but they are fairly easy to understand and configure:
 
-### Vertex Shader
+- the **Input Assembler** is responsible for processing the vertices in an eventual vertex buffer into the primitive topology of our choice, which in our case is a form of triangles, and sending this processed output to the Vertex Shader to process. 
+- The **Output Merger** is responsible for combining the values written by the pixel shader, may that be depth, color or other things, into the one or more render targets that we provide to the OM, we only have one render target for now.
 
-The **Vertex Shader** is the stage where our vertices are processed however we want, although we don't do much processing here, and in the end they're transformed to [screen-space coordinates](link to the coordinate system chapter?)
+The **Vertex Shader** is the stage where our vertices are processed however 
+we want, although we don't do much processing here, and in the end they're 
+transformed to [screen-space coordinates](link to the coordinate system chapter?)
 
-The vertices are usually read from a **Vertex Buffer** which are laid out in a particular way. The vertex shader will be run however many times we tell it to run, which is specified in the first parameter of `ID3D11DeviceContext::Draw()` (more on this later), for instance if we call `Draw(3, 0)`, the vertex shader will run 3 times.
+The vertices are usually read from a **Vertex Buffer** which are laid out in 
+a particular way. The vertex shader will be run however many times we tell 
```

Comment:
Maybe mention the possibilities of not doing it like this without saying exactly how (because stuff like Vertex Pulling exists);
> The vertices are usually read from a **Vertex Buffer** which are laid out in 
a particular way.

->

"Though not required specifically, the easiest method for retrieving vertices is from a **Vertex Buffer**, this is a buffer containing all vertices laid out in a specific way. There are other more advanced methods but we'll leave those out for the sake of simplicity for now."

Reply:
I agree that usually is not emphasizing how when programming on the GPU, "the world is [still] your oyster", I still want to hint at the broad ways you can do things if we do this change though-

The vertices are **usually** read from a **Vertex Buffer** — usually since there are no
hard rules in programming, we can always be inventive to create or read data differently on the fly!
This all works since the vertex shader will be run however many times we tell 
it to run, which is specified in the first parameter of `ID3D11DeviceContext::Draw()` 
(more on this later), for instance if we call `Draw(3, 0)`, the vertex shader will 
run 3 times.

Reply:
I can also reorder the full screen triangle example this way